opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -176,6 +176,7 @@ message Metric {
     // This field will be removed in ~3 months, on July 1, 2021.
     IntHistogram int_histogram = 8 [deprecated = true];
     Histogram histogram = 9;
+    ExponentialHistogram exponential_histogram = 12;
```

Comment:
Curious, why is this one not following the number ordering?

Reply:
oh, I didn't realize--I've re-used number 10 so the fields are in order. (Was 10 ever used?)